src/plugins/data_source/config.ts

```diff
@@ -41,7 +41,7 @@ export const configSchema = schema.object({
     ),
   }),
   clientPool: schema.object({
-    size: schema.number({ defaultValue: 5 }),
+    size: schema.number({ defaultValue: 10 }),
```

Comment:
Does this value means we keep 10 active connections to opensearch cluster?

Reply:
at most 10, it has mechanism to clean up stale ones